firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedFid.java

```diff
@@ -21,37 +21,38 @@
 import com.google.firebase.FirebaseApp;
 
 /**
- * A layer that locally caches a few Firebase Installation attributes on top the Firebase
- * Installation backend API.
+ * A layer that locally persists a few Firebase Installation attributes on top the Firebase
+ * Installation API.
  */
-public class FiidCache {
-  // Status of each cache entry
-  // NOTE: never change the ordinal of the enum values because the enum values are stored in cache
+public class PersistedFid {
+  // Status of each presisted fid entry
+  // NOTE: never change the ordinal of the enum values because the enum values are stored in shared
+  // prefs
   // as their ordinal numbers.
-  public enum CacheStatus {
-    // Cache entry is synced to Firebase backend
+  public enum PersistedStatus {
+    // Persisted Fid entry is synced to Firebase backend
     REGISTERED,
-    // Cache entry is waiting for Firebase backend response or internal network retry
+    // Persisted Fid entry is waiting for Firebase backend response or internal network retry
     UNREGISTERED,
-    // Cache entry is in error state when syncing with Firebase backend
+    // Persisted Fid entry is in error state when syncing with Firebase backend
     REGISTER_ERROR,
-    // Cache entry is in delete state before syncing with Firebase backend
+    // Persisted Fid entry is in delete state before syncing with Firebase backend
     DELETED
   }
 
-  private static final String SHARED_PREFS_NAME = "FiidCache";
+  private static final String SHARED_PREFS_NAME = "PersistedFid";
 
   private static final String FIREBASE_INSTALLATION_ID_KEY = "Fid";
   private static final String AUTH_TOKEN_KEY = "AuthToken";
   private static final String REFRESH_TOKEN_KEY = "RefreshToken";
   private static final String TOKEN_CREATION_TIME_IN_SECONDS_KEY = "TokenCreationEpochInSecs";
```

Comment:
This is usually called a "Timestamp".
Any reason to prefer "Creation Time in Seconds" over "Timestamp"?